cmd/cosign/cli/dockerfile/verify.go

```diff
@@ -82,6 +85,20 @@ func getImagesFromDockerfile(dockerfile io.Reader) ([]string, error) {
 }
 
 func getImageFromLine(line string) string {
+	if strings.HasPrefix(strings.ToUpper(line), "COPY") {
```

Comment:
If I were to pass in a Dockerfile like:
```
FROM golang:1.18 AS builder
RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .

FROM alpine:latest 
WORKDIR /root/
COPY --from=builder /go/src/github.com/alexellis/href-counter/app .
CMD ["./app"]  
```

it would fail. `builder` is not an image reference, but it will be successfully passed as one with `name.ParseReference` Maybe it would be good to add a unit test to cover this scenario 😄

Reply:
I am pretty sure also that the value after the `AS` could be any kind of string as well (e.g. `build1`), so will make it hard to try and interpret at the `--from` level

Reply:
Hmmm, actually I expect two image references: `golang:1.18` and `alpine:latest`

>  it will be successfully passed

But this is the expected case? I couldn't get why you expect this would fail. 🤔 

I added this case into unit tests anyway.

Reply:
So after checking out your branch and testing out, I see that I misunderstood and the test case I presented is fine (as reflected in the test case you added). However, the following scenario:

```
COPY --from=nginx:latest /etc/nginx/nginx.conf /nginx.conf
```

results in the `nginx:latest` not getting pinned to a digest.

Reply:
Hey @ChaosInTheCRD, just pushed a new commit now it supports to resolve the following statement:

```
COPY --from=nginx:latest /etc/nginx/nginx.conf /nginx.conf
```

Thanks!